### PR TITLE
Feat/menu-navigation

### PR DESCRIPTION
**Pull request by IgboPharaoh**
WIP

### CODE REVIEW

**Comment by adamjonas** [comment]
The newsletter page should look more like the landing page but we can handle that in a follow up post.

**Comment by IgboPharaoh** [comment]
> Mobile looks good
> 
> Why does the rss feed link trigger a download?

I've tried to replicate the rss issue on my end and I've not been able to even after testing with the latest deployment. Please can you outline the steps you took to arrive at this issue?

**Comment by adamjonas** [comment]
I just navigated to the rss feed on my phone. Not a blocker if no one else is running into it.

**Comment by adamjonas** [comment]
let's get code review from @Extheoisah or @Emmanuel-Develops so we can 🚢 it

**Comment by IgboPharaoh** [comment]
> I just navigated to the rss feed on my phone. Not a blocker if no one else is running into it.

Alright

File: src/app/newsletter/page.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  return (
+    <div>
+      <NewsletterPage newsletter={newsletters} />
+      <section>
+        <h2 className='text-xl md:text-4xl font-normal pb-8 pt-10'>Monthly Newsletters </h2>
+        <div className='flex flex-col gap-5'>
+          {newsletter_sets.map((set, index) => (
+            <>
+              <div key={`${index}_${set.week_0.link}`}>
```
Comment (Emmanuel-Develops) [inline src/app/newsletter/page.tsx:68]: any reason we are showing only the first two entries `week_0` and `week_1`

for example, the november newsletter set has 4 newsletter entries

**Review by Emmanuel-Develops — CHANGES_REQUESTED** [review]

**Comment by Extheoisah** [comment]
There's a hydration mismatch error when I run the code
image

There's another warning about updating the nextjs version because the one we have is outdated.
I have rebased and updated the static folder causing the conflict.

**Comment by Extheoisah** [comment]
> There's a hydration mismatch error when I run the code image
> 
> There's another warning about updating the nextjs version because the one we have is outdated. I have rebased and updated the static folder causing the conflict.

I'll raise another pr to update nextjs. This hydration error is non-blocking by the way. We can merge.

**Pull request merged**